tests/unittests/JITTest.cpp

```diff
@@ -88,6 +88,20 @@ TEST(JITCorrectnessTest, convTest) {
   EXPECT_TRUE(out1.isEqual(out2));
 }
 
+TEST(JITCorrectnessTest, extract3Dtest) {
+  Tensor inputs(ElemKind::FloatTy, {10, 100, 100});
```

Comment:
nit: The test only uses indices 0 through 3 for slicing in the 1st dimension in inputs. Maybe it's intentional to not use the all of the input's 1st dimension, but if so you could just make this `{5, 100, 100}` and accomplish the same thing.